plant/plants/api/views.py

```diff
@@ -30,3 +33,21 @@ class WateringViewSet(viewsets.ModelViewSet):
     permission_classes = [IsAuthenticated, IsStaffAndCanWatering]
     serializer_class = PlantWateringSerializer
     queryset = Watering.objects.all()
+
+    def perform_create(self, serializer):
+        serializer.save()
+        plant = serializer.validated_data["plant"]
+        Notification.objects.create(
+            recipient=plant.owner,
+            text=f"Plant {plant.name} is now watering",
+            created_by=self.request.user,
+        )
+        message = {
```

Comment:
logic from the lines L45-L49 should be extracted to the send method of Notification model